.github/workflows/windows-mavenwrapper-build.yml

```
@@ -0,0 +1,24 @@
+name: Windows Maven Wrapper Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: Windows Java ${{ matrix.java }} Maven Wrapper
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        java: [11]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Build Project
+        run: ./mvnw.cmd clean install
```

Do we need `./` on Windows

```suggestion
        run: mvnw.cmd clean install
```

?

---

I tried this but workflow fails when I don't add `./`:
```
Run mvnw.cmd clean install
mvnw.cmd: D:\a\_temp\d8391a79-e7aa-498f-8996-6e39eb828a92.ps1:2
Line |
   2 |  mvnw.cmd clean install
     |  ~~~~~~~~
     | The term 'mvnw.cmd' is not recognized as a name of a cmdlet, function, script file, or executable
     | program. Check the spelling of the name, or if a path was included, verify that the path is correct
     | and try again.
```

---

I got it. It's because the call of `mvnw.cmd` is performed from PowerShell script. My PowerShell suggests me to use `.\mvnw.cmd`, but I'm OK with `./mvnw.cmd`.